### PR TITLE
Reject people from people.xml

### PR DESCRIPTION
**Pull request by g4s8**
#494 - remove people from xml on reset

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#667](https://github.com/zerocracy/farm/issues/667) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#667](https://github.com/zerocracy/farm/issues/667) is assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/591356071). The budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4). Please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @yegor256[/z](https://www.0crat.com/u/yegor256) (the architect) right in this ticket. If you decide that this PR should not be accepted ever, also inform the architect.

**Comment by carlosmiranda** [comment]
@yegor256 looks good to merge

File: src/main/resources/com/zerocracy/stk/pmo/awards/reject_people_below_score.groovy [hunk 1]
```diff
@@ -34,9 +35,14 @@ def exec(Project project, XML xml) {
...
-    // @todo #390:30min We should remove people from people.xml
-    //  when their score goes below 200. Let's implement that.
-    //  We should also notify the person in case that happens.
+    People people = new People(farm).bootstrap()
+    people.remove(login)
+    claim.copy()
+      .token("user;${login}")
+      .param(
+        'message',
+        new Par('Your reputation becomes too low, so you have been disconnected from your mentor as in §44').say()
```
Comment (yegor256) [inline src/main/resources/com/zerocracy/stk/pmo/awards/reject_people_below_score.groovy]: @g4s8 you are saying that we remove the mentor, but actually removing the user

**Comment by yegor256** [comment]
@g4s8 the logic is wrong. We should not remove the user entirely, we should do what the "breakup" is doing, but without a penalty. We should just remove the mentor. The user remains in the system, but will need to be invited again.

**Comment by g4s8** [comment]
@yegor256 
> the logic is wrong. We should not remove the user entirely

but it was asked in #494 issue:
> We should remove people from people.xml 

how should I deal with it as XDSD developer?

**Comment by yegor256** [comment]
@g4s8 if this is not critical, you can puzzle it and say "fix it in the future". If it's critical and we can not allow this to appear in production, you should fix it. There is no other way around.

**Comment by g4s8** [comment]
@yegor256 fixed, please see new changes

**Comment by yegor256** [comment]
@rultor merge

**Comment by g4s8** [comment]
@yegor256 please restart Rultor, it's `Address already in use` error again

**Comment by yegor256** [comment]
@rultor merge

**Comment by 0crat** [comment]
Order was finished, quality was "acceptable": +15 points just awarded to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda)

**Comment by 0crat** [comment]
The job [#667](https://github.com/zerocracy/farm/issues/667) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 points just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256)

**Pull request merged**